src/main/java/org/fungover/mmotodo/team/TeamController.java

```diff
@@ -0,0 +1,85 @@
+package org.fungover.mmotodo.team;
+
+import jakarta.validation.Valid;
+import org.fungover.mmotodo.team.Team;
+import org.fungover.mmotodo.team.TeamDto;
+import org.fungover.mmotodo.team.TeamService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.graphql.data.method.annotation.Argument;
+import org.springframework.graphql.data.method.annotation.MutationMapping;
+import org.springframework.graphql.data.method.annotation.QueryMapping;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import java.util.List;
+
+@Controller
+public class TeamController {
+    private final TeamService teamService;
+
+    @Autowired
+    public TeamController(TeamService teamService) {
+        this.teamService = teamService;
+    }
+
+    @QueryMapping
+    public Team teamById(@Argument int id) {
+        return teamService.getTeamById(id);
+    }
+
+    @QueryMapping
+    public List<Team> allTeams() {
+        return teamService.getAllTeams();
+    }
+
+
+    @MutationMapping
+    @ResponseStatus(HttpStatus.CREATED)
```

Comment:
Normally graphql endpoint should always return 200 OK and any status messages are put into the returned json objekt as attributes, so we can remove responsestatus. I made an issue #33 about this asking for refactoring of the existing controllers.

Reply:
ok